### DISCUSSION TITLE
RG350 Amiga (LHA) Files aren´t working

### DISCUSSION THREAD

**Discussion by gautschh** [eduardofilo/RG350_adam_image#95] [General]
regarding V1.2: #30 NES-Games are working fine, but Amiga Games (shown up now): nothing starts after Kickstart ROM message (existing)  

A600 Kickstart problem on my side --> you can close this issue, great work @eduardofilo this image is a great upgrade for the RG350 handheld

**Comment by eduardofilo** [comment]
No problem. That microcomputer systems are rarely plug and play.
